### ISSUE TITLE
Remove deprecated state positioner function blocks

### ISSUE THREAD

**Issue by klauer** [pcdshub/lcls-twincat-motion#145]
It's not a major rework to upgrade to the actually supported versions, so these should be removed in the next major release.

https://github.com/pcdshub/lcls-twincat-motion/tree/master/lcls-twincat-motion/Library/POUs/Deprecated

* `FB_EpicsInOut.TcPOU`
* `FB_PositionStateManager.TcPOU`
* `FB_StatePTPMove.TcPOU `

(Anyone disagree?)

**Comment by klauer** [comment]
These are now obsolete by a couple versions. Should we carry them forward @ZLLentz ?

**Comment by ZLLentz** [comment]
Yes, I think these should probably be removed (in general: this repo has no "remove the deprecated thing" schedule right now).
